packages/web-twig/src/Resources/components/Collapse/README.md

```diff
@@ -4,45 +4,41 @@ This is Twig implementation of the [Collapse] component.
 
 Basic example usage:
 
-```html
-<button type="button" role="button" data-toggle="collapse" data-target="CollapseExample">trigger</button>
-<div id="CollapseExample" class="Collapse">
-  <div class="Collapse__content">content</div>
-</div>
+```twig
+<Button data-toggle="collapse" data-target="CollapseExample">Collapse trigger</Button>
+<Collapse id="CollapseExample">Collapse content</Collapse>
 ```
 
 Usage with link:
 
-```html
-<a role="button" href="javascript:void(0)" data-toggle="collapse" data-target="CollapseExample"> trigger </a> ...
+```twig
+<ButtonLink href="javascript:void(0)" data-toggle="collapse" data-target="CollapseExample">Collapse trigger</ButtonLink> ...
 ```
 
 Open on load example (by aria-expanded):
 
-```html
-<button ... aria-expanded="true">trigger</button> ...
+```twig
+<Button ... aria-expanded="true">Collapse trigger</Button> ...
 ```
 
 Open on load example (by class):
 
-```html
-<button ... class="is-expanded">trigger</button> ...
+```twig
+<Button ... class="is-expanded">Collapse trigger</Button> ...
 ```
 
 Responsive usage for tablet
 
-```html
+```twig
 ...
-<button ... class="d-tablet-none">trigger</button>
-<div id="CollapseExample" class="Collapse" data-breakpoint="tablet">
-  <div class="Collapse__content">...</div>
-</div>
+<Button ... class="d-tablet-none">Collapse trigger</Button>
```

Comment:
I've noticed it before and this is the closest change next to which I can mention that: It would be nice to have all these examples valid Twig/HTML/React. Benefits:

1. working/better syntax highlighting when rendered,
2. possibility of making the examples live in future.

Haven't tried, but this is what I mean:

```suggestion
<Button {# … #} class="d-tablet-none">Collapse trigger</Button>
```

Reply:
Ok, fixed that and I will be aware of this request in future PRs.